flow/raster_cache.h

```diff
@@ -22,7 +22,7 @@ class RasterCacheResult {
 
   virtual ~RasterCacheResult() = default;
 
-  virtual void draw(SkCanvas& canvas, const SkPaint* paint = nullptr) const;
+  virtual void draw(SkCanvas& canvas, const SkPaint* paint) const;
```

Comment:
You can also overload draw with a non-virtual method that just takes in a "canvas" if you want the user code to look the same.

Reply:
There were only a couple of call sites, so I left it as is.